File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.py
from typing import List

class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        a = [0] * 100        
        for i in nums:       
            a[i] += 1
        
        b = []
        for i in range(len(a)):  
            if a[i] == 2:
                b.append(i)
        
        return b

File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        map<int,int> mpp;
        vector<int> v;

        for(auto i : nums){
            mpp[i]++;
        }
        for(auto it : mpp){
            if(it.second>1){
                v.push_back(it.first);
            }
        }
        return v;


    }
};